trin-core/src/jsonrpc/service.rs

```diff
@@ -84,13 +84,17 @@ pub fn launch_jsonrpc_server(
 fn set_ipc_cleanup_handlers(ipc_path: &str) {
     {
         let ipc_path = ipc_path.to_string();
-        ctrlc::set_handler(move || {
+        if let Err(err) = ctrlc::set_handler(move || {
             if let Err(err) = fs::remove_file(&ipc_path) {
                 debug!("Ctrl-C: Skipped removing {} because: {}", ipc_path, err);
             };
             std::process::exit(1);
-        })
-        .expect("Error setting Ctrl-C handler.");
+        }) {
+            warn!(
```

Comment:
Have you experienced dangling ipc sockets when running the peertest? They're not the end of the world, it's a pretty obvious debug when trying to restart trin with an unavailable socket. But, it does seem like the warning could pass by pretty quickly on the output, especially if you're connecting with bootnodes immediately.

Reply:
I haven't yet. For better or worse, we seem to have lots of failsafes (when catching the panic, or at the end of the serving loop).

Reply:
Maybe the other thing to note is that this should be quite unusual to run multiple instances of trin in the same parent process.